obspy/io/mseed/core.py

```diff
@@ -495,6 +495,12 @@ def allocate_data(samplecount, sampletype):
     del lil  # NOQA
     return Stream(traces=traces)
 
+# Define helper function to copy data, replaces use of: trace.data.copy().astype(dtype)
```

Comment:
PEP8: 2 blank lines before and after function definitions (see CircleCI log)

Also, this comment should instead be a docstring..

```
def ...():
    """
    .....  <- here
    """
```
```